girder/models/upload.py

```diff
@@ -305,7 +308,8 @@ def createUploadToFile(self, file, user, size, reference=None,
         return self.save(upload)
 
     def createUpload(self, user, name, parentType, parent, size, mimeType=None,
-                     reference=None, assetstore=None, attachParent=False):
+                     reference=None, assetstore=None, attachParent=False,
+                     saveUpload=True):
```

Comment:
We should document this parameter. I'm also inclined to just call it `save`, since that's what we call it in most other places.